src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -111,6 +113,10 @@ const RFC5280_CRITICAL_CA_EXTENSIONS: &[asn1::ObjectIdentifier] =
 const RFC5280_CRITICAL_EE_EXTENSIONS: &[asn1::ObjectIdentifier] =
     &[BASIC_CONSTRAINTS_OID, SUBJECT_ALTERNATIVE_NAME_OID];
 
+pub enum PolicyError {
+    Other(&'static str),
```

Comment:
We're not using the other variants yet so I've just added `Other` for the time being.

Reply:
Makes sense, thanks for calling it out!